static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -57,7 +50,6 @@ function Breadcrumbs() {
     ref: listRef,
     deps,
   });
-  const {handleDimensionChange} = useVirtualListDimentionChange({cache, listRef});
```

Comment:
I think we still want this, it just got renamed to `useVirtualListDimensionChange`